jenkins2/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+jenkins2:
+  build: .
+  environment:
+    JAVA_OPTS: "\
+      -Djava.awt.headless=true \
+      -Djenkins.install.runSetupWizard=false \
+      -Dorg.apache.commons.jelly.tags.fmt.timeZone=Europe/London \
+      -Duser.timezone=Europe/London \
+      -Duser.country=GB "
+  ports:
+    - "50000:50000"
+    - "8080:8080"
```

Comment:
enabling ssl for the elb -> jenkins connection is in the unwritten backlog. please make it secure by default